public/app/features/plugins/sandbox/custom_formatter.ts

```diff
@@ -0,0 +1,279 @@
+// Original code from https://github.com/salesforce/near-membrane/blob/main/packages/near-membrane-dom/src/custom-devtools-formatter.ts
```

Comment:
This whole file is taken from near-membrane directly "as is". It offers the functionality to inspect the proxy objects in chrome dev console. Otherwise only a `Proxy()` object would print.

Reply:
This file is also only loaded in development mode.